lib/model/QubitActor.php

```diff
@@ -648,4 +669,135 @@ public static function setTermRelationByName($name, $options = array())
 
     return $relation;
   }
+
+  /****************
+   Import methods
+  *****************/
+  /**
+   * Wrapper for QubitDigitalObject::importFromUri() method
+   *
+   * @param array $uris URIs of remote files
+   * @return QubitActor $this
+   *
+   * @TODO allow for different usage types
+   */
+  public function importDigitalObjectFromUri($uris, &$errors)
```

Comment:
Given this code (and importDigitalObjectFromBase64, etc.) is very similar to QubitInformationObject::importDigitalObjectFromUri maybe we can make a Redmine issue, for the future, to make this shared (in a child class of QubitObject or somewhere).

Reply:
I agree.  Will see that tickets are created.